### DISCUSSION TITLE
App wont launch?

### DISCUSSION THREAD

**Discussion by ARealFool** [GuidodiPasquo/Actively-Stabilized-Model-Rocket-Simulator#5] [General]
Im currently using Python version 3.11.2. I installed all the required libraries and even the optional one. Yet, when it tried to run the program, it opens and quickly closes the console. When I tried to run it via the cmd, it just returns an os error. Is there something I'm missing?

**Comment by GuidodiPasquo** [comment]
I just tested it and got no error, could you paste it here?

**Reply by ARealFool** [reply]
apologies for the late reply. Here's everything that was deisplayed on my shell:
 `Microsoft Windows [Version 10.0.19044.2728]
(c) Microsoft Corporation. All rights reserved.

C:\Users\windows 10\Desktop\Rocketry\AeroVECTOR-master>python -m AeroVECTOR.py
Traceback (most recent call last):
  File "<frozen runpy>", line 189, in _run_module_as_main
  File "<frozen runpy>", line 112, in _get_module_details
  File "C:\Users\windows 10\Desktop\Rocketry\AeroVECTOR-master\AeroVECTOR.py", line 45, in <module>
    os.chdir(os.path.dirname(sys.argv[0]))
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: ''

C:\Users\windows 10\Desktop\Rocketry\AeroVECTOR-master>
`
